include/revng/Pipes/PipelineManager.h

```diff
@@ -127,7 +127,12 @@ class PipelineManager {
 
   /// A helper function used to produce all possible targets. It is used for
   /// debug purposes to see if any particular target crashes.
-  llvm::Error produceAllPossibleTargets(bool ExpandTargets = true);
+  llvm::Error produceAllPossibleTargets() {
+    return produceAllPossibleTargets(false);
+  }
+  llvm::Error produceAllPossibleSingleTargets() {
+    return produceAllPossibleTargets(true);
+  }
```

Comment:
This change doesn't seem to belong here. Can you squash it in the commit introducing `bool ExpandTargets`?

Reply:
isolated in a comit of its own